Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM docker.io/library/alpine:3.13
+
+ARG network
+ENV IXO_HOME=/home/ixo/
+
+RUN apk add --no-cache ca-certificates libgcc libstdc++ tzdata
+
+COPY bin/ixod /usr/bin/ixod
+
+COPY ${network}/genesis.json /ixo/genesis.json
+COPY entrypoint.sh /ixo/entrypoint.sh
+
+RUN apk add --no-cache ca-certificates libc6-compat
```

Comment:
`ca-certificates` is being installed twice - in both line 6 and line 13
Additionally, both `apk add` can be combined to reduce number of layers in the resulting Docker image